### PR TITLE
Remove honeycomb from makefile

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

Saw this was in the Makefile still. Also fixed some formatting.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -346,28 +346,25 @@ webserver_test: bin/rds-combined-ca-bundle.pem server_generate mocks_generate bi
...
-  SERVE_DPS=true \
-  SERVE_API_INTERNAL=true \
-  SERVE_API_PUBLIC=true \
-  MUTUAL_TLS_ENABLED=true \
+	SERVE_SDDC=true \
+	SERVE_ORDERS=true \
+	SERVE_DPS=true \
+	SERVE_API_INTERNAL=true \
+	SERVE_API_PUBLIC=true \
+	MUTUAL_TLS_ENABLED=true \
```
Comment (chrisgilmerproj) [inline Makefile]: Makefiles need tabs not spaces.

**Comment by mr337** [comment]
I think there is a dep or something broken as my PR failed too for something unrelated.

**Comment by chrisgilmerproj** [comment]
> I think there is a dep or something broken as my PR failed too for something unrelated.

Where did yours fail? on the Orders API? There may have been something hidden and broken due to the way spaces were used instead of tabs. Would be nice to suss out.

**Comment by mr337** [comment]
> > I think there is a dep or something broken as my PR failed too for something unrelated.
> 
> Where did yours fail? on the Orders API? There may have been something hidden and broken due to the way spaces were used instead of tabs. Would be nice to suss out.

It was at the OrderAPI IIRC. Oddly enough mine is now passing but having issues running this PR locally that doesn't seem to be related to the PR itself.

**Comment by chrisgilmerproj** [comment]
> It was at the OrderAPI IIRC. Oddly enough mine is now passing but having issues running this PR locally that doesn't seem to be related to the PR itself.

Oddly same thing happened to me. Passing locally but only fails on CircleCI on Orders API.  I'll rerun and see what happens. There's no reason this ought to fail based on the tab/spaces change.

**Comment by chrisgilmerproj** [comment]
I had to rerun the tests 3 times. Failed on different API endpoint tests each time. Not super happy about the flaky tests but at least its not this code change that's the problem.

**Pull request merged**